### ISSUE TITLE
OS mining services needs resource deposit crate

### ISSUE THREAD

**Issue by Oskutin** [loiste-interactive/Obenseuer-Issues#3111]

**Comment by Nextej** [comment]
Alternatively (and arguably more cool idea) would be to have a marked "Unloading Zone" infront of the OS mining services where you can park your own storage wagons, that automatically connect with the renovator's inventory
![image](https://github.com/user-attachments/assets/e54e3038-1def-4135-b4b6-2aa547fe9915)

**Comment by Orphiex** [comment]
> Alternatively (and arguably more cool idea) would be to have a marked "Unloading Zone" (with paint or hazard tape) infront of the OS mining services where you can park your own storage wagons, that automatically connect with the renovator's inventory

It's a cool idea, and I definitely want a tenement resources box in the mines, but at the moment the OS Mining Services only offer basement upgrades; none of the upgrades require ores or other mine materials, just surface ones, so the "Unloading Zone" doesn't offer much benefit for this purpose. You'd still have to run the materials down to the mines and load them into the cart first; at best, it cuts a minor portion of running.

On the other hand, it could be a neat addition to the upgraded tenement mine depot. Especially once you've bought a rock crusher or two and started making gravel/limestone.

TLDR: Resources box yes, unloading zone maybe much later.

**Comment by Nextej** [comment]
@Orphiex you can find a resource chest on a flatbed wagon (as well as other cargo wagons), with the tenement mine depo (once it will work), it will be easier to move resource from the tenement to the train.
